src/components/v5/common/MemberSignature/MemberSignature.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+
+import { useIntl } from 'react-intl';
+import UserAvatar from '~v5/shared/UserAvatar';
+import { MemberSignatureProps } from './types';
+
+const displayName = 'v5.common.MemberSignature';
+
+const MemberSignature: FC<MemberSignatureProps> = ({ user, isSigned }) => {
+  const { formatMessage } = useIntl();
+
+  const { name, profile } = user;
+  const { displayName: userDisplayName } = profile || {};
+
+  return (
+    <div className="flex items-center justify-between">
+      <UserAvatar user={user} size="xs" userName={userDisplayName || name} />
```

Comment:
Why do you need to pass userName, if you're passing the whole user object?

Reply:
Then add a TODO

Reply:
Or it would be better to update it now. If it takes more time to change this than create a new UserAvatar component then please create a new one. We will replace it in other places later